spring-cloud-contract-spec/src/main/groovy/org/springframework/cloud/contract/spec/internal/HandlebarsContractTemplate.groovy

```diff
@@ -0,0 +1,69 @@
+package org.springframework.cloud.contract.spec.internal
+
+import groovy.transform.CompileStatic
+import org.springframework.cloud.contract.spec.ContractTemplate
+
+/**
+ * Represents the structure of templates using Handlebars compatible with
+ * WireMock template model requirements.
+ *
+ * @author Marcin Grzejszczak
+ * @since 1.1.0
+ */
+@CompileStatic
+class HandlebarsContractTemplate implements ContractTemplate {
+
+	@Override
+	String openingTemplate() {
+		return "{{{"
+	}
+
+	@Override
+	String closingTemplate() {
+		return "}}}"
+	}
+
+	@Override
+	String url() {
+		return wrapped("request.url")
+	}
+
+	@Override
+	String query(String key) {
+		return wrapped("request.query.${key}.[0]")
```

Comment:
could call the overloaded version here to reduce some magic string duplication, but that's just a style thing